src/arrays/programs_involving_multidimensional_data/medium/q4/OddorEven.java

```diff
@@ -0,0 +1,52 @@
+package arrays.programs_involving_multidimensional_data.medium.q4;
+
+/*
+Write a Java program that initializes a 2D array of random integers of size 5 by 5, with numbers in the range of 1 to 10
+(inclusive of both values), and print each row in a line. Then use a method named count that creates a 1D array
+with two elements, and count the number of even and odd numbers in the 2D array. Use the index 0 for even numbers and
+index 1 for the odd numbers. Then, print out the resulting 1D array.
+
+Sample Output:
```

Comment:
I think we should add some additional print statements to the output. Maybe something like:
```
Generated Array:
    10 2 2 7 10
    7 6 1 3 6
    4 1 3 10 3
    9 5 8 5 4
    3 4 1 6 5
Number of even numbers: 12
Number of odd numbers: 13
```